deliver/spec/commands_generator_spec.rb

```diff
@@ -211,7 +211,7 @@ def expect_download_screenshots_run_with(expected_options)
 
       expect(Deliver::Runner).to receive(:new) do |actual_options|
         expect(expected_options._values).to eq(actual_options._values)
-        # ugly work-around to do the work that DetectValues would normally do
+        # ugly workaround to do the work that DetectValues would normally do
```

Comment:
there are more cases to avoid hyphenations, here's one where it's proper to use `workaround` (n)